### DISCUSSION TITLE
How to set TLS cert path on kafka events

### DISCUSSION THREAD

**Discussion by EKrol2** [minio/minio#20261] [Q&A] [answered]
Hi, Im trying to setup up a replication event to kafka from minio using TLS. However when I use the command, (or web interface) i get an error saying that the file cannot be found.

> mc: <ERROR> Unable to set 'notify_kafka:NAME brokers=BROKER_LIST topic=TOPIC_NAME tls=on client_tls_cert=PATH_TO_P12 client_tls_key=KEY' to server: error (NAME:kafka): open PATH_TO_P12: no such file or directory.

Im using the following command:
`mc admin config set az1/ notify_kafka:NAME brokers="BROKER_LIST" topic="TOPIC_NAME" tls="on" client_tls_cert="PATH_TO_P12" client_tls_key="KEY"`

I've tried with a path to the p12 file on my local machine, and have tried putting the cert in a bucket and providing that path, however both result in the same error.

I am using the minio cli from my local machine, so not the machine that minio runs on, should the certificates be added to the filesystem minio itself uses? And if so can i do this from the outside using the minio cli?

Thanks!

**Comment by harshavardhana** [comment, marked as answer]
First mount the certs as config map on the pods, and then provide that path here in the config.

**Reply by EKrol2** [reply]
Hi @harshavardhana , thanks for the response.

We do not have access to the kubernetes cluster / namespace where MinIO is deployed since we are merely a tenant and we are adding our own certificates all our kafka topics. So we cannot add a configmap that the MinIO deployment itself can use.
